packages/core/src/abi/common/index.ts

```diff
@@ -0,0 +1,8 @@
+export const buffLength = (buf: string) => (buf.length - 2) / 2
```

Comment:
Could just be `common.ts` file.

Can you add links or explanation of what's going on in here, as much as possible?